core/domain/summary_services.py

```diff
@@ -222,3 +223,26 @@ def get_featured_exploration_summary_dicts(language_codes):
         reverse=True)
 
     return _get_displayable_exp_summary_dicts(sorted_exp_summaries)
+
+def get_top_rated_exploration_summary_dicts(language_codes):
+    """Returns a list of top rated explorations with the given language code.
+
+    The return value is sorted in decreasing order of search rank.
+    """
+    filtered_exp_summaries = [
+        exp_summary for exp_summary in
+        exp_services.get_non_private_exploration_summaries().values()
+        if exp_summary.language_code in language_codes]
+
+    search_ranks = {
+        exp_summary.id: exp_services.get_search_rank_from_exp_summary(
```

Comment:
Actually... is it the search rank you want, or the average rating?

Maybe sort instead by (average rating, number of ratings); if two average ratings are equal, prioritize the one with the higher number of ratings.